src/stores/applets.js

```diff
@@ -123,6 +132,14 @@ export const useAppletsStore = defineStore('applets', () => {
             if (!map[page]) map[page] = {}
             if (!map[page][area]) map[page][area] = []
             map[page][area].push(applet)
+          } else if (Array.isArray(area.location)) {
+            // If the area is an object that include location info,
+            // make it accessible at that location
+            area.location.forEach((localArea) => {
+              if (!map[page]) map[page] = {}
+              if (!map[page][localArea]) map[page][localArea] = []
+              map[page][localArea].push(applet)
+            })
```

Comment:
It took me a minute to see the problem that `location` is trying to solve - namely within resourceDetailsTabs, how can you specify the tab order. Is that right?

Stated generally, it's an optional third level hierarchy (after page and area) of where exactly in the application the target is. It's a good idea and implemented well, I think! I think what's tripping me up is that `location` (and `id` in CbApplet Target) are kinda vague names and are different from one-another. Could you rename them something more specific and both the same? My ideas for "something more specific than area" were `subarea`, `section`, `block`, or `element`, but I'd be up for whatever you think is logical.

Definitely shout if I'm misunderstanding something here.

Reply:
id is now "appletId" to make it clearer, and `location` is now `position`

Reply:
And I wrote this as a longer comment the first time but it seems to have disappeared

Reply:
My thoughts being, this is supposed to work with/around how tabs work in the CUI, where it uses the label and resource type the same way, but adds them onto the end of the tabs. so having the new `position` indicate where the tabs are located and works as an addition to the existing info. (I feel like the stuff you suggested like subarea, section, block, or element are good for specificity but imply there's more flexibility than there actually is on the HUI tabs rendering)